### PR TITLE
fix CD scripts so instances launch with the correct key pair

### PR DESCRIPTION
**Pull request by macdiesel**
@edx/pipeline-team PTAL.

### CODE REVIEW

File: playbooks/continuous_delivery/launch_instance.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     ec2:
       instance_tags: {"Name" : "{{ automtion_prefix }} {{ unique_key_name.stdout }}"}
       region: "{{ ec2_region }}"
-      key_name: "{{ unique_key_name.stdout }}"
+      key_name: "{{ automtion_prefix }} {{ unique_key_name.stdout }}"
```
Comment (doctoryes) [inline playbooks/continuous_delivery/launch_instance.yml]: Nit: It's totally anal - but could `automtion` be globally replaced to `automation`?

**Comment by doctoryes** [comment]
👍  with a small nit

File: playbooks/continuous_delivery/launch_instance.yml (hunk 1 above)
Comment (macdiesel) [inline playbooks/continuous_delivery/launch_instance.yml]: yes we can!

**Comment by doctoryes** [comment]
🚀 🚢 🇮🇹

**Pull request merged**